HermaFx.DataAnnotations/MinElementsIfAttribute.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace HermaFx.DataAnnotations
+{
+	public class MinElementsIfAttribute : RequiredIfAttribute
+	{
+		private const string _defaultErrorMessage = "There should be at least {0} elements.";
```

Comment:
Try to use MinElementsAttribute internally, so the message & logic dont need to be duplicated.